lib/matplotlib/tests/test_contour.py

```diff
@@ -158,7 +158,7 @@ def test_contour_manual_labels():
 
 @image_comparison(baseline_images=['contour_labels_size_color'],
                   extensions=['png'], remove_text=True)
-def test_contour_manual_labels():
+def test_contour_labels_size_color():
 
```

Comment:
Updated. Surprisingly, that's the only test that had that problem. The other newly activated tests pass, thankfully.